src/contained-list/contained-list-story.module.ts

```diff
@@ -0,0 +1,31 @@
+import { NgModule } from "@angular/core";
+import { IconService } from "carbon-components-angular/icon";
+import Apple16 from "@carbon/icons/es/apple/16";
+import Apple20 from "@carbon/icons/es/apple/20";
+import Fish16 from "@carbon/icons/es/fish/16";
+import Fish20 from "@carbon/icons/es/fish/20";
+import Strawberry16 from "@carbon/icons/es/strawberry/16";
+import Strawberry20 from "@carbon/icons/es/strawberry/20";
+import SubtractAlt16 from "@carbon/icons/es/subtract--alt/16";
+import Wheat16 from "@carbon/icons/es/wheat/16";
+import Wheat20 from "@carbon/icons/es/wheat/20";
+import { CommonModule } from "@angular/common";
+
+@NgModule({
+	imports: [CommonModule]
+})
+export class ContainedListStoryModule {
+	constructor(private iconService: IconService) {
+		this.iconService.registerAll([
+			Apple16,
+			Apple20,
+			Fish16,
+			Fish20,
+			Strawberry16,
+			Strawberry20,
+			SubtractAlt16,
+			Wheat16,
+			Wheat20
+		]);
+	}
+}
```

Comment:
Could you move this file under a `stories` directory. Have a look at the table component for reference

Would be something like:
```
.
└── stories/
    └── contained-list-story.module.ts
```

Also, do we need to import & register the size 20 icons? I think we are using only size 16.

Reply:
Done on [cdc3821](https://github.com/carbon-design-system/carbon-components-angular/pull/2464/commits/cdc38211338c179aba4f3ff630c434de4f6ca46a)